tests/sentry/auth/test_superuser.py

```diff
@@ -189,6 +193,30 @@ def test_su_access_no_request(self, logger):
                 "superuser.logged-in", extra={"ip_address": "127.0.0.1", "user_id": 10}
             )
 
+    @freeze_time("2022-03-03 08:21:34")
+    def test_expired_check_org_in_request(self):
+        request = self.build_request(expires=self.current_datetime)
+        superuser = Superuser(request, allowed_ips=())
+        assert superuser.is_active is False
+        assert not getattr(request, "organization", None)
```

Comment:
i feel like a test where superuser would be active otherwise would be a better test for this behavior (outside of org change limit but otherwise active with no org in the request)